app/components/Header.tsx

```diff
@@ -53,14 +56,14 @@ export default function Header() {
             Request a Quote
           </ChakraLink>
 
-          <IconButton aria-label="Hamburger menu" onClick={handleHamburger}>
-            <Image
-              src="/images/hamburger-menu.png"
-              alt="Hamburger Menu"
-              width={40}
-              height={40}
-            />
-          </IconButton>
+          <IconButton
+            aria-label="open navigation menu"
+            onClick={handleHamburger}
+            variant="ghost"
+            color="white"
+            fontSize="50px"
+            icon={<FaBars />}
+          />
```

Comment:
Just noticed that when you click on the hamburger menu button, it turns gray (I think it's default color in ChakraUI theme for active buttons or something). So we might need to override default values for that in theme or directly in the component.

![Screen Shot 2024-02-01 at 3 23 24 PM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/6e22a7a4-1b9f-467f-917c-b84538239f12)